charts/identity-providers-config/values.yaml

```
@@ -0,0 +1,19 @@
+
+# This is to override the chart name.
+nameOverride: ""
+fullnameOverride: ""
+
+
+services:
+  externalSecretsOperator:
+    defaultClusterSecretStore: vault-hub-secret-store
+
+environmentShort: ~
```

As par our agreement, dependencies need to be explicitly defined. Please mention in the comment, that global parameter is used as a default.

```suggestion
environmentShort: ~ # if not specified, global.apc.environmentShort is used
```